module2/homework/transformContainers/transform.cpp

```diff
@@ -0,0 +1,29 @@
+#include "transform.hpp"
+
+#include <algorithm>
+#include <iostream>
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(
+    std::list<std::string> uniqueList,
+    std::deque<int> uniqueDeque) {
```

Comment:
Misleading indentations.

Reply:
Agree, but this is from the auto-format. I'll check if this formating can be changed in clang-format.
Nevetheless, correction provided :)